views/retentionReasons.njk

```diff
@@ -0,0 +1,158 @@
+{% extends "./partials/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = 'Prevent removal of this record' %}
+{% set subtitle = 'Reason for preventing removal' %}
+{% set instructions = 'Select all that may apply. Remove all to stop retaining the record.' %}
+
+{% block content %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: offenderUrl
+  }) }}
+
+  <div class="govuk-grid-row govuk-body">
+    <div class="govuk-grid-column-full">
+
+      <div class="basic-details">
+        <div class="govuk-!-margin-right-6">
+          <img
+                  src="/app/images/{{ offenderBasics.offenderNo }}/data"
+                  alt="Headshot of {{ offenderBasics.firstName }} {{ offenderBasics.lastName }}"
+                  style="float:left"
+                  height="110"
+                  width="90"
+                  data-qa="offender-image"
+          />
+        </div>
+  
+        <div>
+          <p class="offender-details-paragraph bold">Name</p>
+          <span class="offender-details-paragraph"
+                data-qa="displayName">{{ offenderBasics.lastName }}, {{ offenderBasics.firstName }}</span>
+        </div>
+  
+        <div>
+          <p class="offender-details-paragraph bold">Prison No.</p>
+          <span class="offender-details-paragraph"
+                data-qa="nomsId">{{ offenderBasics.offenderNo }}</span>
+        </div>
+  
+        <div>
+          <p class="offender-details-paragraph bold">Date of Birth</p>
+          <span class="offender-details-paragraph" data-qa="dob">{{ offenderBasics.dateOfBirth }}</span>
+        </div>
+  
+        <div>
+          <p class="offender-details-paragraph bold">Location</p>
+          <span class="offender-details-paragraph" data-qa="prison-description">{{ agency }}</span>
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row govuk-body">
+    <div class="govuk-grid-column-full">
+
+      <h1 class="govuk-heading-l">{{ title }}</h1>
+
+      {% set otherHtml %}
+      {{ govukTextarea({
+        id: "more-detail",
+        name: "more-detail",
+        classes: "govuk-!-width-one-third",
+        label: {
+          text: "Please give details"
+        }
+      }) }}
+      {% endset -%}
+
+      {{ govukCheckboxes({
+        idPrefix: "citizenship-conditional",
+        name: "citizen",
+        fieldset: {
+          legend: {
+            text: subtitle,
+            classes: "govuk-fieldset__legend--m"
+          }
+        },
+        hint: {
+          text: instructions 
+        },
+        items: [
```

Comment:
should these be pulled from the data compliance service?

Reply:
just thinking that if you wanted to change these that would only be a release to data compliance not here

Reply:
👍 yes that's the end goal, but just wanted to start with something that visually matches what I need